test/RyanJsonRFC8259JsonTest.c

```
@@ -0,0 +1,613 @@
+
+		char aaa[300] = {0};
+		snprintf(aaa, sizeof(aaa), "%s/%s", path, name);
+
+		FILE *f = fopen(aaa, "rb");
+		if (f == NULL) { goto fail; }
+
+		fseek(f, 0, SEEK_END);
+		long len = ftell(f);
+		fseek(f, 0, SEEK_SET);
+		char *data = (char *)malloc(len + 10);
+		fread(data, 1, len, f);
```

Cppcheck (2.18.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)